openforcefield/utils/toolkits.py

```diff
@@ -3438,6 +3442,360 @@ def assign_partial_charges(
 
         molecule.partial_charges = charges * unit.elementary_charge
 
+    @classmethod
+    def _elf_is_problematic_conformer(
+        cls, molecule: "Molecule", conformer: unit.Quantity
+    ) -> Tuple[bool, Optional[str]]:
+        """A function which checks if a particular conformer is known to be problematic
+        when computing ELF partial charges.
+
+        Currently this includes conformers which:
+
+        * contain a trans-COOH configuration. These conformers ... TODO add reason.
```

Comment:
@davidlmobley I think you understand the reason for this a little better than me - would you be able to write a one liner rationalising the removal of trans COOH conformers?

Reply:
I happened to stumble across this digging through old issues, but I didn't think it was ever implemented? https://github.com/openforcefield/openforcefield/issues/428

Reply:
@mattwthompson discarding the COOH trans configuration is done by default by OE, so on the OE side this is implemented. The difference between this implementation and the OE one is that instead of raising an exception, OE now tries to generate a cis conformation if one is not present. While this could likely also be done for RDKit, it is beyond the scope of this PR.

Why the trans conformer should be discarded goes a bit beyond the reason mentioned in the issue I think, as the cis conformer is the electrostatically most interacting conformer as it has the internal h-bond which is not present in the trans conformer.

Reply:
@SimonBoothroyd sorry I'd missed this initially. See for example here: https://github.com/openforcefield/openforcefield/issues/428

In terms of a one-liner, as I recall, it goes this way: The trans conformer is discarded because it leads to strong electrostatic interactions when assigning charges, and these result in unreasonable charges. (Downstream calculations have observed up to a 4 log unit error in water-octanol logP calculations when using charges assigned from trans conformers, which AmberTools allows.)

Reply:
Thanks!